server/frontend/src/components/Comments/PublicationForm.vue

```diff
@@ -0,0 +1,492 @@
+          :entry="entry"
+          v-on:update-filename="entry.testcase = $event"
+          v-on:update-content="testCaseContent = $event"
+        />
+
+        <div v-if="createError" class="alert alert-danger" role="alert">
+          <button
+            type="button"
+            class="close"
+            data-dismiss="alert"
+            aria-label="Close"
+            v-on:click="createError = null"
+          >
+            <span aria-hidden="true">&times;</span>
+          </button>
+          An error occurred while submitting the comment to
+          <strong>{{ provider.hostname }}</strong
+          >: {{ createError }}
+        </div>
+
+        <div
+          v-else-if="createdCommentId"
+          class="alert alert-success"
+          role="alert"
+        >
+          Comment
+          <strong>
+            <a :href="commentLink" target="_blank">{{ createdCommentId }}</a>
+          </strong>
+          was successfully submitted and created on
+          <strong>{{ provider.hostname }}</strong> provider.
+        </div>
+
+        <div
+          v-if="publishCrashDataError"
+          class="alert alert-danger"
+          role="alert"
+        >
+          <button
+            type="button"
+            class="close"
+            data-dismiss="alert"
+            aria-label="Close"
+            v-on:click="publishCrashDataError = null"
+          >
+            <span aria-hidden="true">&times;</span>
+          </button>
+          An error occurred while attaching crash data to the created external
+          comment: {{ publishCrashDataError }}
+        </div>
+
+        <div
+          v-if="publishTestCaseError"
+          class="alert alert-danger"
+          role="alert"
+        >
+          <button
+            type="button"
+            class="close"
+            data-dismiss="alert"
+            aria-label="Close"
+            v-on:click="publishTestCaseError = null"
+          >
+            <span aria-hidden="true">&times;</span>
+          </button>
+          An error occurred while attaching the testcase to the created external
+          comment: {{ publishTestCaseError }}
+        </div>
+
+        <div v-if="!bugzillaToken" class="alert alert-warning" role="alert">
+          Please define an API Token for
+          <strong>{{ provider.hostname }}</strong> in your settings to submit a
+          new comment.
+        </div>
+
+        <div>
+          <button
+            type="button"
+            class="btn btn-danger"
+            v-on:click="createExternalComment"
+            :disabled="!bugzillaToken || submitting"
+          >
+            Submit
+          </button>
+          <button
+            type="button"
+            class="btn btn-default"
+            v-on:click="goBack"
+            :disabled="submitting"
+          >
+            Cancel
+          </button>
+        </div>
+      </form>
+    </div>
+  </div>
+</template>
+
+<script>
+var Mustache = require("mustache");
```

Comment:
Just curious but why not import?

Reply:
I guess we could use import, I just stupidly c/c the Mustache doc: [https://github.com/janl/mustache.js/#usage](https://github.com/janl/mustache.js/#usage)

Do you want me to update that?